packages/dds/sequence/src/revertibles.ts

```diff
@@ -147,7 +147,7 @@ function revertLocalDelete(
 	const end = string.localReferencePositionToPosition(revertible.end);
 	const intType = revertible.interval.intervalType & ~IntervalType.Simple;
 	const type = intType | IntervalType.SlideOnRemove;
-	const props = revertible.interval.properties;
+	const { intervalId, ...props } = revertible.interval.properties;
```

Comment:
Add a comment here that it's necessary to strip out the id so that the revert creates a new interval id rather than reusing which leads to eventual consistency bugs.